docs/CHANGELOG.asciidoc

```diff
@@ -87,6 +88,16 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+[[release-notes-5.6.13]]
+== {kib} 5.6.13
+coming[5.6.13]
+
+[float]
+=== Enhancement
+
+Operations::
+* Upgrades the version of ChromeDriver to 2.42.1 {pull}23648[#23648]
+
```

Comment:
That PR is really only for the test framework so I don't know if we should list it in release notes?  I'll leave it up to you.  It doesn't hurt anything.